doc/design/volumes/nfsvolumestores.md

```diff
@@ -70,33 +70,53 @@ The advantage to using the interface is the storage layer maintains consistency
  48 }
 ```
 
-When we create the NFS `VolumeStore`, we'll store the NFS target parameters (`host` + `path`) in the implementation's struct.  This is the only information we'll need to mount the NFS target on the container.
+When we create the NFS `VolumeStore`, we'll store the NFS target parameters (`host` + `path`) in the implementation's struct.  This is the only information we'll need to mount the NFS target on the container. NFS based volume stores will work for the default volume store as well.
 
 ```
-type NFSv3VolumeStore struct {
- target *url.URL
+
+// VolumeStore this is nfs related volume store definition
```

Comment:
`// VolumeStore stores nfs-related volume store information` or something to that effect.